src/ipc/ipc4/handler.c

```diff
@@ -49,10 +49,10 @@ struct ipc4_msg_data {
 	uint32_t delayed_error;
```

Comment:
@lyakh usually we do one patch per logical change. Here you have 4). but I guess they are simple enough.